cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-settings-step/UnmanagedClustersSettingsAdvanced.tsx

```diff
@@ -0,0 +1,129 @@
+// React imports
+import React, { ChangeEvent, useContext, useState } from 'react';
+
+// Library imports
+import { ClarityIcons, blockIcon, blocksGroupIcon, clusterIcon } from '@cds/core/icon';
+import { CdsControlMessage } from '@cds/react/forms';
+import { CdsInput } from '@cds/react/input';
+import { useForm } from 'react-hook-form';
+
+// App imports
+import { INPUT_CHANGE } from '../../../../state-management/actions/Form.actions';
+import { StepProps } from '../../../../shared/components/wizard/Wizard';
+
+ClarityIcons.addIcons(blockIcon, blocksGroupIcon, clusterIcon);
+
+interface FormInputs {
+    CLUSTER_NAME: string;
+    CONTROL_PLANE_NODE_COUNT: string;
+    WORKER_NODE_COUNT: string;
+}
+
+const unmanagedClusterProviders = [
+    {
+        label: 'calico',
+        value: 'CALICO',
+    },
+    {
+        label: 'anthrea',
+        value: 'ANTHREA',
+    },
+    {
+        label: 'none',
+        value: 'NONE',
+    },
+];
+function UnmanagedClusterSettingsAdvanced(props: Partial<StepProps>) {
+    const { handleValueChange, currentStep } = props;
+    const {
+        register,
+        formState: { errors },
+    } = useForm<FormInputs>();
+
+    const [selectedProvider, setSelectedProvider] = useState('KIND');
+
+    const handleClusterNameChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
ditto previous remark on multiple handlers when one would suffice

Reply:
I will Make sure to do this but I think that I should leave that for the advanced unmanaged cluster wizard since this pr was supposed to be focused on simple/basic config (most of the code in advanced is not staying there it was a placeholder to better see the switch)